hub/core/tensor.py

```diff
@@ -200,6 +256,77 @@ def _check_array_and_tensor_are_compatible(tensor_meta: dict, array: np.ndarray)
         )
 
 
+def read(image_path: str, check_meta: bool = True):
```

Comment:
Why load from a file instead of a standard image format? Is the intended syntax:
```
ds.image.append(None, read("path_to_image.png"))
```
? I think we can do much better, especially since this is such an important operation.

Reply:
It was decided to have the replacement of the PIL Image loader with similar syntax, so that image will be opened only if check_meta=True. Agree about the append syntax, will change it to have one parameter.